### DISCUSSION TITLE
Edge Update

### DISCUSSION THREAD

**Discussion by PhatBread** [furybee/chrome-tab-modifier#392] [General]
Is anyone else having issues with the extension on the latest version of Microsoft Edge? I heard it may have been from an API update on Edge's part, might there be an update coming for it?

**Comment by mackenziejonesosit** [comment]
I have this issue too, seems the 2 new versions havent been pushed to the Edge Store yet and we are still on the broken 1.0.11 version

**Reply by sylouuu** [reply]
1.0.13 Edge version is online!

**Reply by mackenziejonesosit** [reply]
Saw it this morning and already making use of it again, thank you for restoring organisation to my work life!

**Reply by PhatBread** [reply]
I believe in Christmas miracles again, thank you!
